src/App.js

```diff
@@ -22,33 +22,38 @@ const App = () => {
   const [hand, setHand] = useState([])
 
   const drawCard = () => {
-    // 1)
+    // 1) This variable holds a randomized value from the arrays suit and rank, it will randomly select an index with the Math function. 
     const draw = `${rank[Math.floor(Math.random() * rank.length)]} ${
       suit[Math.floor(Math.random() * suit.length)]
     }`
-    // 2)
+    // 2) The if statement which has a condition of the setter hook "hand" index must be strictly equal to -1, which means no cards have been drawn yet.
```

Comment:
This is looking at whether the randomly created card already exists in the hand. If it does already exist the corresponding block of code won't run.